### PR TITLE
Update grub files order to keep same with 1.x branch

### PR DESCRIPTION
**Pull request by chenlizhong**
Signed-off-by: lichen <lichen@redhat.com>

### CODE REVIEW

File: insights/combiners/grub_conf.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def __init__(self, grub1, grub2, grub2_efi, grub1_efi):
         self.version = self.is_efi = self.is_kdump_iommu_enabled = None
         self.grub = self.kernel_initrds = None
         # get grub configuration
-        self.grub = grub1 or grub2 or grub2_efi or grub1_efi
+        self.grub = grub1 or grub1_efi or grub2 or grub2_efi
```
[the remark this answers is not in the document]
Comment (xiangce) [reply insights/combiners/grub_conf.py]: @lonicerae - any thoughts?

**Comment by chenlizhong** [comment]
@xiangce 
Yes, It is best to get the currently using grub file. Here is just hard code the order.
Yesterday, I open one card for this  https://trello.com/c/6MLqAEIF
@lonicerae Please investigate it.

**Comment by chenlizhong** [comment]
@xiangce 
I get another idea. We can merge this PR first. To sync 3.0 and 1.0.  Then use another PR to enhance the combiner. This can make the commit clear.

**Comment by xiangce** [comment]
@chenlizhong - make sense to me, going to merge this PR af first.

**Pull request merged**